packaging/rpm/templates/copyright

```diff
@@ -0,0 +1,8 @@
+Comment: Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
RPM doesnt need a copyright file like this. The convention for RPM is to install any LICENSE files shipped by upstream but no extra/custom copyright file like this.